doc/source/ray-core/actors/terminating-actors.rst

```diff
@@ -64,7 +64,42 @@ Ray to :ref:`automatically restart <fault-tolerance-actors>` the actor, make sur
 flag ``no_restart=False`` to ``ray.kill``.
 
 For :ref:`named and detached actors <actor-lifetimes>`, calling ``ray.kill`` on
-an actor handle will destroy the actor and allow the name to be reused.
+an actor handle destroys the actor and allow the name to be reused.
+
+From Ray 2.0 onwards, you should be able to view the death cause of an actor in `death_cause` as part of the actor state by querying for information about the actor, with the :ref:`State API <state-api-overview-ref>`, or see the Actors View in the :ref:`Dashboard <observability-getting-started>`.
```

Comment:
"use `ray list actors --detail <link to the page>` to see a list of dead actors and the reason".

```
# This API is only available when Ray is downloaded by `pip install "ray[default]"`
ray list actors
```

```
  ---
  -   actor_id: e8702085880657b355bf7ef001000000
      class_name: Actor
      state: DEAD
      job_id: '01000000'
      name: ''
      node_id: null
      pid: 0
      ray_namespace: dbab546b-7ce5-4cbb-96f1-d0f64588ae60
      serialized_runtime_env: '{}'
      required_resources: {}
      death_cause:
          actor_died_error_context: # <---- You could see the error message w.r.t why the actor exits. 
              error_message: The actor is dead because `ray.kill` killed it.
              owner_id: 01000000ffffffffffffffffffffffffffffffffffffffffffffffff
              owner_ip_address: 127.0.0.1
              ray_namespace: dbab546b-7ce5-4cbb-96f1-d0f64588ae60
              class_name: Actor
              actor_id: e8702085880657b355bf7ef001000000
              never_started: true
              node_ip_address: ''
              pid: 0
              name: ''
      is_detached: false
      placement_group_id: null
      repr_name: ''
```